packages/resources/src/util/apiGatewayV1AccessLog.ts

```diff
@@ -0,0 +1,133 @@
+import * as cdk from "@aws-cdk/core";
+import * as logs from "@aws-cdk/aws-logs";
+import * as apig from "@aws-cdk/aws-apigateway";
+import { App } from "../App";
+
+export interface AccessLogProps extends apig.CfnStage.AccessLogSettingProperty {
+  retention: logs.RetentionDays; 
+}
+
+export type AccessLogData = {
+  logGroup: logs.LogGroup | undefined,
+  format: apig.AccessLogFormat,
+  destination: apig.LogGroupLogDestination,
+}
+
+export type AccessLogDestinationConfig = {
```

Comment:
Didn't notice `AccessLogDestinationConfig` used elsewhere, let's not export.

Reply:
No it isn't. Updated.